### PR TITLE
factore update header

### PR DESCRIPTION
**Pull request by arichelle**
Update of fastCore.m to be compatible with createTissueSpecificModel, update header and remove duplicate

### CODE REVIEW

**Comment by arichelle** [comment]
I've try to remove the duplicate (src/dataIntegration/transcriptomics/fastcore.m) using devTools but it does not want to commit the deletion...

**Comment by laurentheirendt** [comment]
Fixes issue #515.

**Comment by laurentheirendt** [comment]
@arichelle, thanks. I removed the file now. What exactly was the issue with the devTools?

Would you also mind formatting the documentation here? @jacekwachowiak had done such a nice job already. A documentation guide is [here](https://github.com/opencobra/cobratoolbox/blob/master/docs/source/guides/documentationGuide.md). Thanks.

File: src/dataIntegration/transcriptomics/FASTCORE/fastcore.m [hunk 1]
```diff
@@ -1,65 +1,51 @@
...
+    if nargin < 3 || isempty(epsilon)
+        epsilon=1e-4;
     end
 
+    coreSetRxn = core;    
     model_orig = model;
 
     %Find irreversible reactions
-    irrevRxns = find(model.lb>=0); %% could be called irrevRxns
+    irrevRxns = find(model.rev==0);
```
Comment (laurentheirendt) [inline src/dataIntegration/transcriptomics/FASTCORE/fastcore.m]: @arichelle, I think that @tpfau deprecated `model.rev` recently in PR #507. Also, in PR #513, @tpfau already had fixed this specific line. 

May you revert back to what he had changed? Thanks.

Comment (arichelle) [inline src/dataIntegration/transcriptomics/FASTCORE/fastcore.m]: Sorry didn't catch that

**Pull request merged**